pallets/moonbeam-lazy-migrations/src/lib.rs

```diff
@@ -51,6 +52,26 @@ pub mod pallet {
 	/// The total number of suicided contracts that were removed
 	pub(crate) type SuicidedContractsRemoved<T: Config> = StorageValue<_, u32, ValueQuery>;
 
+	#[pallet::storage]
+	/// The total number of suicided contracts that were removed
+	pub(crate) type StateMigrationStatus<T: Config> = StorageValue<_, MigrationStatus, ValueQuery>;
+
+	pub(crate) type StorageKey = BoundedVec<u8, ConstU32<512>>;
+
+	#[derive(Clone, Encode, Decode, scale_info::TypeInfo, PartialEq, Eq, MaxEncodedLen)]
+	pub enum MigrationStatus {
```

Comment:
```suggestion
	pub enum StateMigrationStatus {
```

Reply:
It would conflict with the storage name

Reply:
Then add Storage suffix in the storage name for example.
Anyway, this palelt lazy migrations is for all lazy migrations, so each type name should specify for what migration it is

Reply:
Oh got it